nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeo.conf

```diff
@@ -10,8 +10,10 @@
 #JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
 #JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_152.jdk/Contents/Home
 
-## Heap size
-JAVA_OPTS=-Xms512m -Xmx1024m
+## If you run Nuxeo in a Linux container, the JVM will automatically detect the Control Group memory limit with the UseContainerSupport option, enabled by default.
+## You can then control the memory with the following options: InitialRAMPercentage, MaxRAMPercentage and MinRAMPercentage.
+## By default, we set the maximum heap size to 50 % of the cgroup memory limit.
+JAVA_OPTS=-XX:MaxRAMPercentage=50
```

Comment:
This is a production configuration, we used to set a minimal memory footprint by default which is useful on dev env especially if you have 32GB of ram.

Reply:
Yes, reworked.